flink-table/flink-sql-jdbc-driver-bundle/pom.xml

```diff
@@ -39,11 +39,19 @@
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-sql-jdbc-driver</artifactId>
 			<version>${project.version}</version>
+			<scope>provided</scope>
```

Comment:
IIUC, `scope` is unnecessary when we use shade plugin with 'include' (a.k.a, white list). See also 'flink-table-api-java-uber'.